exporter/awscloudwatchlogsexporter/config.go

```diff
@@ -77,41 +83,13 @@ func (config *Config) Validate() error {
 	if config.QueueSettings.QueueSize < 1 {
 		return errors.New("'sending_queue.queue_size' must be 1 or greater")
 	}
-	if !isValidRetentionValue(config.LogRetention) {
-		return errors.New("invalid value for retention policy.  Please make sure to use the following values: 0 (Never Expire), 1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, 2192, 2557, 2922, 3288, or 3653")
+	if retErr := cwlogs.ValidateRetentionValue(config.LogRetention); retErr != nil {
+		return retErr
 	}
-	return nil
-}
-
-// Added function to check if value is an accepted number of log retention days
-func isValidRetentionValue(input int64) bool {
-	switch input {
-	case
-		0,
-		1,
-		3,
-		5,
-		7,
-		14,
-		30,
-		60,
-		90,
-		120,
-		150,
-		180,
-		365,
-		400,
-		545,
-		731,
-		1827,
-		2192,
-		2557,
-		2922,
-		3288,
-		3653:
-		return true
+	if tagInputErr := cwlogs.ValidateTagsInput(config.Tags); tagInputErr != nil {
+		return tagInputErr
 	}
-	return false
+	return nil
```

Comment:
This can be simplified to just return the result of validating the tags since there is no further action to take.

```go
return cwlogs.ValidateTagsInput(config.Tags)
```

Reply:
Changed ti simplify the return in my latest commit.